conf/services/conflationTypes.json

```diff
@@ -84,5 +84,16 @@
         "matcher": "hoot::ScriptMatchCreator,LinearWaterway.js",
         "merger": "hoot::ScriptMergerCreator",
         "members": {}
+    },
+    "Differential": {
+        "matcher": "",
+        "merger": "",
+        "members": {
+            "differential.remove.reference.data": "Remove reference data from differential output layer",
```

Comment:
@bwitham and or @curranMapper - are the descriptive names (right of the semi-colon) ok? better if changed?